### ISSUE TITLE
input placeholder is unable to recognize / deal with more text than input provides without scrolling

### ISSUE THREAD

**Issue by honestbleeps** [lorenzoongithub/completely#27]
The input placeholder is misaligned if the user has typed enough into the text input that text starts scrolling over, because the placeholder position calculation doesn't account for that.

for example, set a width of 100px to the text input on the draw a robot demo and start typing something like "paint nose aqua" - http://complete-ly.appspot.com/examples/draw.a.robot.html 

it may be rare for many users of this library (which is fantastic, by the way), but I'm going to need to come up with a solution somehow. Is this something you've considered / encountered?

**Comment by lorenzoongithub** [comment]
@honestbleeps I agree. (although This is a kind of edge case) it would be nice to have it fixed. 
Maybe in an upcoming version if I get some time to work on it.

**Comment by honestbleeps** [comment]
I may see what I can do about committing a fix back for you. I can't promise, but it's kind of important for me to get fixed so I'm going to at least tinker.  The catch is that my only use case / interest is modern non-IE browsers only, so I'm not going to be able to dedicate time to testing IE, etc. I don't see a browser compatibility promise on immediate review of your site so I'm not sure if that's OK or not!

**Comment by honestbleeps** [comment]
oh, odd, I can't submit a PR because the code isn't actually on github :-)

any particular reason for that?

**Comment by lorenzoongithub** [comment]
@honestbleeps if you solve it, just let me know how you did it and I will consider it for the next release.(if i ever get a chance to do it).  thanks

**Comment by sandyplace** [comment]
@honestbleeps did you ever solve this? Would you mind sharing your solution?

**Comment by honestbleeps** [comment]
I actually didn't get the chance / free time to solve it unfortunately, sorry.
